mmaction/__init__.py

```diff
@@ -1,3 +1,15 @@
+import mmcv
+from mmcv import digit_version
+
 from .version import __version__, short_version
 
+mmcv_minimum_version = '1.1.1'
+mmcv_maximum_version = '1.2'
```

Comment:
upper bound a dependency is not a good idea

Reply:
Agree.